distribution/docker/solr/README.md

```diff
@@ -0,0 +1,32 @@
+# DDF Compatible Solr Container
+
+This docker image simply builds on top of the official solr image and adds in a few components to make it fully compatible with DDF
+
+## Usage
+
+To use this image, run: `docker container run -d -p 8994:8983 codice/ddf-solr:<ddf-version>`
+
+## Extending
+
+For scenarios where additional cores, plugins, or libaries are required, this image can be used as a source via `FROM codice/ddf-solr:<ddf-version>`
+
+To add an additional ddf compatible core(s) a helper script has been included, run `create-ddf-core [cores...]` in a downstream `Dockerfile`
+
+# Details
+
+## Added plugins
+
+* `ddf.platform.solr:solr-xpath`
+
+## Added libraries
+
+* `org.locationtech.jts:jts-core`
+
+## Added helper scripts
+
+* `create-ddf-core [coresName ...]`
+
+
+# Pushing to docker.io
```

Comment:
❓ Do we have to push the image to docker.io? Will anything break if the `docker.io` entry is missing.

Reply:
Suggest providing an example entry for docker.io in settings.xml. That way, people can copy and paste into their own settings file.

Reply:
If we don't push it anywhere it won't be usable outside the context of this projects maven build. Just like we push our maven artifacts to nexus, docker images belong in a docker registry.

I'll add an example of the necessary settings in here and make sure to note that it only is required when running the deploy phase